dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -596,6 +615,17 @@ public Organization updateOrganization(@ApiParam(hidden = true) @Parameter(hidde
             }
         }
 
+        // If user is a global admin, allow a transition between HIDDEN and APPROVED statuses.
```

Comment:
If the user is not an admin and tries this, they should get an unauthorized error... Now it's just silently ignoring.

Reply:
Gotchya.  If an admin tries to make an illegal state transition, say from HIDDEN to PENDING, should they get an error, versus a silent ignore?  There's precedent for silently ignoring in the org creation code, status is clamped to PENDING.

Reply:
I would say an error.

I see your point about org creation, and it's a little bit of a gray area, e.g., there are many other fields a user could specify in a create, e.g., id, dbCreateDate and dbUpdateDate that will (or at least should) get ignored.

But this one feels different; it's one of the fields we let the user change, and if it's not valid, it should throw an error, IMO.